salt/key.py

```diff
@@ -501,7 +501,13 @@ def check_minion_cache(self, preserve_minions=None):
             if os.path.isdir(m_cache):
                 for minion in os.listdir(m_cache):
                     if minion not in minions and minion not in preserve_minions:
-                        shutil.rmtree(os.path.join(m_cache, minion))
+                        try:
+                            shutil.rmtree(os.path.join(m_cache, minion))
+                        except (OSError, IOError) as ex:
+                            log.warning("Key: Delete cache for %s got OSError/IOError: %s \n",
```

Comment:
Could you please change from double-quotes here to single-quotes?

Reply:
OK，Thank you so much